cmd/mcp-broker/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+)
+
+func main() {
+	var (
+		controllerMode = flag.Bool("controller", false, "Run in controller mode")
+		configFile     = flag.String("config", "", "Path to broker configuration file")
```

Comment:
prob can rebase on mine? as it has all of this in it?

Reply:
I think I'll remove, stick with yours

Reply:
fixed on rebase